dotnetv3/StepFunctions/Actions/StepFunctionsWrapper.cs

```diff
@@ -0,0 +1,283 @@
+        return response.HttpStatusCode == HttpStatusCode.OK;
+    }
+
+    // snippet-end:[StepFunctions.dotnetv3.DeleteActivity]
+
+    // snippet-start:[StepFunctions.dotnetv3.DeleteStateMachine]
+    /// <summary>
+    /// Delete a Step Functions state machine.
+    /// </summary>
+    /// <param name="stateMachineArn">The Amazon Resource Name (ARN) of the
+    /// state machine.</param>
+    /// <returns>A Boolean value indicating the success of the action.</returns>
+    public async Task<bool> DeleteStateMachine(string stateMachineArn)
+    {
+        var response = await _amazonStepFunctions.DeleteStateMachineAsync(new DeleteStateMachineRequest
+        { StateMachineArn = stateMachineArn });
+        return response.HttpStatusCode == HttpStatusCode.OK;
+    }
+
+    // snippet-end:[StepFunctions.dotnetv3.DeleteStateMachine]
+
+    // snippet-start:[StepFunctions.dotnetv3.DescribeExecution]
+    /// <summary>
+    /// Retrieve information about the specified Step Functions execution.
+    /// </summary>
+    /// <param name="executionArn">The Amazon Resource Name (ARN) of the
+    /// Step machines execution.</param>
+    /// <returns>The API response returned by the API.</returns>
+    public async Task<DescribeExecutionResponse> DescribeExecutionAsync(string executionArn)
+    {
+        var response = await _amazonStepFunctions.DescribeExecutionAsync(new DescribeExecutionRequest { ExecutionArn = executionArn });
+        return response;
+    }
+
+    // snippet-end:[StepFunctions.dotnetv3.DescribeExecution]
+
+    // snippet-start:[StepFunctions.dotnetv3.DescribeStateMachine]
+    /// <summary>
+    /// Retrieve information about the specified Step Functions state machine.
+    /// </summary>
+    /// <param name="StateMachineArn">The Amazon Resource Name (ARN) of the
+    /// Step Functions state machine to retrieve.</param>
+    /// <returns>Information about the specified Step Functions state machine.</returns>
+    public async Task<DescribeStateMachineResponse> DescribeStateMachineAsync(string StateMachineArn)
+    {
+        var response = await _amazonStepFunctions.DescribeStateMachineAsync(new DescribeStateMachineRequest { StateMachineArn = StateMachineArn });
+        return response;
+    }
+
+    // snippet-end:[StepFunctions.dotnetv3.DescribeStateMachine]
+
+    // snippet-start:[StepFunctions.dotnetv3.GetActivityTask]
+    /// <summary>
+    /// Retrieve a task with the specified Step Functions activity
+    /// with the specified Amazon Resource Name (ARN).
+    /// </summary>
+    /// <param name="activityArn">The Amazon Resource Name (ARN) of
+    /// the Step Functions activity.</param>
+    /// <param name="workerName">The name of the Step Functions worker.</param>
+    /// <returns>The response from the Step Functions activity.</returns>
+    public async Task<GetActivityTaskResponse> GetActivityTaskAsync(string activityArn, string workerName)
```

Comment:
Could not find this method in the tests.

Reply:
Added test.